config/php/phpcsfixer.rules.yml

```diff
@@ -202,7 +202,7 @@ parameters:
                 - return
                 - switch_case
                 - yield
-        no_unneeded_curly_braces: true
```

Comment:
Is that rule deprecated now?

Reply:
Yes, this is the one in https://github.com/drupol/phpcsfixer-configs-drupal/issues/8:
> - Rule "no_unneeded_curly_braces" is deprecated. Use "no_unneeded_braces" instead.

I deleted the wrong line in the previous PR.